tools/mentors/view_page_text_image.php

```diff
@@ -115,7 +115,7 @@
     echo "<form method='get' action='view_page_text_image.php' target='_top'>\n";
     if(!$project) {
         echo _("Project ID") . ":&nbsp;";
-        echo "<input type='text' maxlength='25' name='projectid' size='25' value='" . attr_safe($projectid) . "'> \n";
+        echo "<input type='text' maxlength='25' name='projectid' size='25' value='" . attr_safe($projectid) . "' required> \n";
```

Comment:
`percent` could also be required, since passing an empty value shows this:

![image](https://user-images.githubusercontent.com/3049847/52264626-6caea800-2932-11e9-826b-24d0f6efef96.png)

https://github.com/DistributedProofreaders/dproofreaders/blob/7b774cfbe8997606fa9efba979bfd3156844a5da/tools/mentors/view_page_text_image.php#L209